src/libraries/Directory.Build.props

```diff
@@ -103,6 +103,14 @@
     <IsPackable Condition="$(MSBuildProjectName.StartsWith('Microsoft.Extensions.')) and '$(IsSourceProject)' == 'true'">true</IsPackable>
   </PropertyGroup>
 
+  <Import Sdk="Microsoft.DotNet.PackageValidation" Project="Sdk.props" Condition="'$(IsPackable)' == 'true' and '$(IsSourceProject)' == 'true'"/>
+
+  <PropertyGroup Condition="'$(IsPackable)' == 'true'">
+    <PackageValidationBaselineVersion>5.0.0</PackageValidationBaselineVersion>
```

Comment:
Please don't hardcode the version here and instead use the library's PackageVersion.Major - 1.